packages/slate-react/src/components/editable.tsx

```diff
@@ -766,7 +767,7 @@ export const Editable = (props: EditableProps) => {
               if (Hotkeys.isRedo(nativeEvent)) {
                 event.preventDefault()
 
-                if (typeof editor.redo === 'function') {
+                if (HistoryEditor.isHistoryEditor(editor)) {
```

Comment:
This change means `history-editor.ts` from `slate-history` is now included within the `slate-react` bundle.